syslog-ng/README.md

```diff
@@ -70,3 +70,14 @@ sudo docker exec -it syslog-ng /bin/bash
 
 ## More information
 For detailed information on how to run your central log server in Docker and other Docker-related syslog-ng use cases, see the blog post [Your central log server in Docker](https://syslog-ng.com/blog/central-log-server-docker/).
+
+
+## Common issues
```

Comment:
I would not call it *issues*, because this is by design, and the result is the only  disabling capability (like setting `--no-caps`).
You could just name it (as you mentioned befor) FAQ.

Reply:
Hi Guys,
the related issue was opened on Jan 18, 2017, you solved it on the same day but 1 and a half year later the docs hasn't been changed a bit
just my opinion (as a user):
the problem is that in the description of the official docker repo (https://hub.docker.com/r/balabit/syslog-ng/) you never mention `--no-cap` or `docker run --cap-add` but there are 4 different `docker run ...` which IMHO useless

Reply:
Dear @levlas,

 first of all: any feedback (including this) is highly appreciated. We, as a developers have a different look on syslog-ng, so things like this might left out from the docs. It is always good to hear back from the community, and we learn a lot from that.
 As for the run commands: I think you are absolutely right. If you have any additional good examples please feel free to send a PR. I'm sure that there are cases that we haven't considered yet.

Br,
Laci

Reply:
Why not just say that "cap_add: - NET_BIND_SERVICE" is needed? The default command shown in the docs for running this container exposes ports that cannot be set by the container without NET_BIND_SERVICE set. Soo...why beat around the bush with "use the --cap-add option"?